main/components/ClipboardCopy/ClipboardCopy.js

```diff
@@ -7,6 +7,32 @@ import styled from 'styled-components'
 import { copyTextToClipboard } from './utils.js'
 
 const StyledButton = styled.button`
+  position:relative;
+  background-color: transparent;
+  background-clip: padding-box;
+  color: ${ props => props.theme.colors.darkRed };
+  cursor: pointer;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  column-gap: 0.5rem;
+  height: 2.2rem;
```

Comment:
Let's change these `rem`s to `em`s. I think these values should depend on the current font size. I know that I put one of them in there, but I'm just re-thinking it.